common/src/Microsoft.Azure.IIoT.Abstractions/src/Diagnostics/IMetricLoggerConfig.cs

```diff
@@ -0,0 +1,18 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
Best to move this interface into the Core package, as it is specific to Application insights.  You should put it into the src/Diagnostics folder there.